.travis.yml

```diff
@@ -19,13 +19,15 @@ matrix:
     env: TOXENV=style
   - python: 3.6
     env: TOXENV=docs
+  - python: 3.7
+    env:
+      - TOXENV=py37
 addons:
-  postgresql: "9.4"
+  postgresql: "10"
```

Comment:
Is this necessary for the upgrade? If we test against Postgres 10, do we no longer officially support 9.4 and anything below < 10? Perhaps I'm reading too much into this, and its really a non-issue?

From our docs - https://dallinger.readthedocs.io/en/latest/developing_dallinger_setup_guide.html#install-postgresql
The lowest version of Postgresql that Dallinger v5 supports is 9.4.
This is fine for Ubuntu 18.04 LTS and 16.04 LTS as they ship with Postgresql 10.4 and 9.5 respectively, however Ubuntu 14.04 LTS ships with Postgresql 9.3